Index: README
===================================================================
RCS file: /cvsroot/smartmontools/sm5/README,v
retrieving revision 1.41
diff -u -p -r1.41 README
--- README	23 Apr 2003 13:24:11 -0000	1.41
+++ README	31 Aug 2003 13:37:57 -0000
@@ -77,6 +77,7 @@ will display the same information.
 
 == INSTALLATION ==
 To install, use the commands:
+  ./configure
   make
   make install
 
Index: smartctl.c
===================================================================
RCS file: /cvsroot/smartmontools/sm5/smartctl.c,v
retrieving revision 1.89
diff -u -p -r1.89 smartctl.c
--- smartctl.c	30 Aug 2003 13:06:48 -0000	1.89
+++ smartctl.c	31 Aug 2003 13:37:58 -0000
@@ -22,6 +22,7 @@
  *
  */
 
+#include "config.h"
 #include <stdio.h>
 #include <unistd.h>
 #include <sys/ioctl.h>
@@ -53,8 +54,7 @@ smartmonctrl *con=NULL;
 long long bytes=0;
 
 void printslogan(){
-  pout("smartctl version %d.%d-%d Copyright (C) 2002-3 Bruce Allen\n",
-      (int)RELEASE_MAJOR, (int)RELEASE_MINOR, (int)SMARTMONTOOLS_VERSION);
+  pout("smartctl version %s Copyright (C) 2002-3 Bruce Allen\n", VERSION);
   pout("Home page is %s\n\n",PROJECTHOME);
   return;
 }
Index: smartd.c
===================================================================
RCS file: /cvsroot/smartmontools/sm5/smartd.c,v
retrieving revision 1.201
diff -u -p -r1.201 smartd.c
--- smartd.c	30 Aug 2003 13:06:48 -0000	1.201
+++ smartd.c	31 Aug 2003 13:38:00 -0000
@@ -20,6 +20,7 @@
  *
  */
 
+#include "config.h"
 #define _GNU_SOURCE
 #include <stdio.h>
 #include <sys/ioctl.h>
@@ -584,8 +585,7 @@ void WritePidFile() {
 
 // Prints header identifying version of code and home
 void PrintHead(){
-  PrintOut(LOG_INFO,"smartd version %d.%d-%d Copyright (C) 2002-3 Bruce Allen\n",
-           (int)RELEASE_MAJOR, (int)RELEASE_MINOR, (int)SMARTMONTOOLS_VERSION);
+  PrintOut(LOG_INFO,"smartd version %s Copyright (C) 2002-3 Bruce Allen\n", VERSION);
   PrintOut(LOG_INFO,"Home page is %s\n\n",PROJECTHOME);
   return;
 }
Index: smartmontools.spec
===================================================================
RCS file: /cvsroot/smartmontools/sm5/smartmontools.spec,v
retrieving revision 1.117
diff -u -p -r1.117 smartmontools.spec
--- smartmontools.spec	30 Aug 2003 13:06:48 -0000	1.117
+++ smartmontools.spec	2 Sep 2003 16:20:31 -0000
@@ -1,4 +1,4 @@
-Release:  19
+Release:  1
 Summary:	SMARTmontools - for monitoring S.M.A.R.T. disks and devices
 Summary(cs):	SMARTmontools - pro monitorování S.M.A.R.T. diskù a zaøízení
 Summary(de):	SMARTmontools - zur Überwachung von S.M.A.R.T.-Platten und-Geräten
@@ -8,7 +8,7 @@ Summary(pt):	SMARTmontools - para monito
 Summary(it):	SMARTmontools - per monitare dischi e dispositivi S.M.A.R.T.
 Summary(pl):	Monitorowanie i kontrola dysków u¿ywaj±æ S.M.A.R.T.
 Name:		smartmontools
-Version:	5.1
+Version:	5.19
 License:	GPL
 Group:		Applications/System
 Group(de):	Applikationen/System
@@ -197,12 +197,20 @@ pochodzi od oprogramowania smartsuite i 
 
 %setup -q
 
+
+
 %build
+%configure
 make
 
 %install
-rm -rf $RPM_BUILD_ROOT
-make DESTDIR=$RPM_BUILD_ROOT install
+rm -rf %{buildroot}
+
+%makeinstall
+#rm -f examplescripts/Makefile*
+
+%clean
+rm -rf %{buildroot}
 
 # Red Hat 6.x: move manuals (man directory) from %datadir to %prefix
 if [ -f /etc/redhat-release ]; then
@@ -219,12 +227,8 @@ fi
 %attr(644,root,root) %{_mandir}/man8/smartctl.8*
 %attr(644,root,root) %{_mandir}/man8/smartd.8*
 %attr(644,root,root) %{_mandir}/man5/smartd.conf.5*
-%doc WARNINGS CHANGELOG COPYING TODO README VERSION smartd.conf examplescripts
+%doc WARNINGS CHANGELOG COPYING TODO README smartd.conf examplescripts
 %config(noreplace) %{_sysconfdir}/smartd.conf
-
-%clean
-rm -rf $RPM_BUILD_ROOT
-rm -rf %{_builddir}/%{name}-%{version}
 
 # The following are executed only by the binary RPM at install/uninstall
 
